sources/light_client.move

```diff
@@ -38,7 +41,7 @@ public fun testnet_params(): Params {
 // default params for bitcoin regtest
 public fun regtest_params(): Params {
     return Params {
-        power_limit: 0x00000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffff,
+        power_limit: 0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff,
```

Comment:
is this correct? I thought in the past PR you fixed this to the orig value

Reply:
OK, found it: https://github.com/bitcoin/bitcoin/blob/v28.1/src/kernel/chainparams.cpp#L523